CHANGES_NEXT_RELEASE

```diff
@@ -2,6 +2,7 @@
 - Add: filtering metadata in notifications (#2507)
 - Fix: removed spurious decimals in metadata numbers in NGSIv1 rendering (a follow up of #2176)
 - Fix: crash when creating entity with metadata when subscription is in place with "mq" on a different metadata (#2496)
+- Fix: correct rendering of metadata compound values when a vector
```

Comment:
I suggest to refer to #1068 here, e.g:

- Fix: correct rendering of metadata compound values when a vector (a bugfix for #1068 functionality)